Common/src/Common/Service/Table/Formatter/InternalConversationMessage.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * Internal conversation message
+ */
+
+namespace Common\Service\Table\Formatter;
+/**
+ * Internal conversation message
+ */
+class InternalConversationMessage implements FormatterPluginManagerInterface
+{
+    /**
+     * status
+     *
+     * @param array $row    Row data
+     * @param array $column Column data
+     *
+     * @return     string
+     * @inheritdoc
+     */
+    public function format($row, $column = null)
+    {
+        if($row['createdBy']['team']) {
```

Comment:
Should some array key exist checking happening here?

Reply:
!empty() would be safe to use here, will update

No null safe operator available yet sadly